lasagne/layers/conv.py

```diff
@@ -9,6 +9,7 @@
 
 
 __all__ = [
+    "conv_output_length",
```

Comment:
maybe not. I added it there to make sure it shows up in the docs, but we can probably work around it if we don't want it in `lasagne.layers`. I'll remove it and see what happens.